source/developer/interactive-messages.rst

```diff
@@ -0,0 +1,329 @@
+.. _interactive-messages:
+
+Interactive Messages
+============================
+
+Mattermost supports interactive messages for `incoming <https://docs.mattermost.com/developer/webhooks-incoming.html>`_ and `outgoing webhooks <https://docs.mattermost.com/developer/webhooks-outgoing.html>`_, `custom slash commands <https://docs.mattermost.com/developer/slash-commands.html>`_ and `plugins <https://docs.mattermost.com/administration/plugins.html>`_ via actions. They help make your integrations richer by completing common tasks inside Mattermost conversations, increasing user engagement and productivity.
+
+Use interactive messages to simplify complex workflows by allowing users to take quick actions directly through your integration post. For example, they enable your integration to:
+
+- mark a task complete in your project management tracker
+- conduct a customer survey or a poll
+- initiate a command to merge a branch into a release
+
+To try it out, you can use this `matterpoll plugin <https://github.com/matterpoll/matterpoll>`_ to add polling to Mattermost channels via a ``/poll`` slash command.
+
+.. image:: ../../source/images/matterpoll.png
```

Comment:
This image isn't rendering for me (I don't think it's in the images folder)

Reply:
Switched back to a gif